deploy/examples/cluster-test.yaml

```diff
@@ -13,6 +13,19 @@ metadata:
   name: my-cluster
   namespace: rook-ceph # namespace:cluster
 spec:
+  # logCollector:
```

Comment:
Are you just testing these changes and planning on reverting? It's nice to keep this cluster-test.yaml brief, and not include comments unless it's commonly needed during development.

Reply:
revert